templates/data/kyma-operator-data.yaml

```diff
@@ -15,7 +15,7 @@ templates:
                   command: "bash"
                   args:
                     - "-c"
-                    - "cd operator && golangci-lint run"
+                    - "cd operator && golangci-lint run --max-issues-per-linter=0 --max-same-issues=0 --sort-results"
```

Comment:
You can also use .golangci.yaml congi inside the repository: https://golangci-lint.run/usage/configuration/
https://github.com/kyma-project/test-infra/blob/main/.golangci.yaml

Reply:
Hi @Halamix2 , thanks for the hint. Yes, we have configuration yaml file in our code base. https://github.com/kyma-project/kyma-operator/blob/main/operator/.golangci.yaml